src/main/java/mServer/crawler/sender/arte/ArteJsonObjectToDatenFilmCallable.java

```diff
@@ -52,52 +55,92 @@ public ArteJsonObjectToDatenFilmCallable(JsonObject aJsonObjec, String aLangCode
-                
-                String beschreibung = getElementValue(programObject, JSON_ELEMENT_KEY_SHORT_DESCRIPTION);
-                
-                String urlWeb = getElementValue(programObject, JSON_ELEMENT_KEY_URL);
-    
-                //https://api.arte.tv/api/player/v1/config/[language:de/fr]/[programId]
-                String programId = getElementValue(programObject, JSON_ELEMENT_KEY_PROGRAM_ID);
-                String videosUrl = String.format(ARTE_VIDEO_INFORMATION_URL_PATTERN, langCode, programId);
-                
-                Gson gson = new GsonBuilder().registerTypeAdapter(ArteVideoDTO.class, new ArteVideoDeserializer()).create();
-    
-                try(Response responseVideoDetails = executeRequest(videosUrl))
+                if(responseVideoDetails.isSuccessful())
                 {
-                    if(responseVideoDetails.isSuccessful())
+                    ArteVideoDTO video = gson.fromJson(responseVideoDetails.body().string(), ArteVideoDTO.class);
+
+                    //The duration as time so it can be formatted and co.
+                    LocalTime durationAsTime = durationAsTime(video.getDurationInSeconds());
+
+                    if (video.getVideoUrls().containsKey(Qualities.NORMAL))
                     {
-                        ArteVideoDTO video = gson.fromJson(responseVideoDetails.body().string(), ArteVideoDTO.class);
+                        String broadcastBegin = ""; 
+                        
+                        OkHttpClient httpClient = MVHttpClient.getInstance().getHttpClient();
+                        //https://api.arte.tv/api/opa/v3/programs/[language:de/fr]/[programId]
+                        String videosUrlVideoDetails2 = String.format(ARTE_VIDEO_INFORMATION_URL_PATTERN_2, langCode, programId);
+                        Request request = new Request.Builder()
+                                .addHeader(MediathekArte_de.AUTH_HEADER, MediathekArte_de.AUTH_TOKEN)
+                                .url(videosUrlVideoDetails2).build();
                         
-                        //The duration as time so it can be formatted and co.
-                        LocalTime durationAsTime = durationAsTime(video.getDurationInSeconds());
-                       
-                        if (video.getVideoUrls().containsKey(Qualities.NORMAL))
+                        try(Response responseVideoDetails2 = httpClient.newCall(request).execute())
                         {
-                            String broadcastBegin = ""; 
-                            
-                            if(jsonObject.has(JSON_ELEMENT_BROADCAST)) {
-                                broadcastBegin = jsonObject.get(JSON_ELEMENT_BROADCAST).getAsString();    
+                          if(responseVideoDetails2.isSuccessful())
+                          {
+                            /*
+                             * Grobe Struktur des Json's:
+                             * {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)